synchronizer/batches.go

```diff
@@ -76,20 +75,19 @@ func NewBatchSynchronizer(
 }
 
 func (bs *BatchSynchronizer) resolveCommittee() error {
-	bs.comitteeLock.Lock()
-	defer bs.comitteeLock.Unlock()
-
-	committee := make(map[common.Address]etherman.DataCommitteeMember)
 	current, err := bs.client.GetCurrentDataCommittee()
 	if err != nil {
 		return err
 	}
-	for _, member := range current.Members {
-		if bs.self != member.Addr {
-			committee[member.Addr] = member
+
+	bs.committee = NewCommitteeMapSafe()
```

Comment:
Nitpick: this is not used until line 90, maybe move it to line 89?

Reply:
Fixed https://github.com/0xPolygon/cdk-data-availability/pull/96/commits/d12a23633b0a097789b17266867024dd8d1f78b7